docs/UserGuide.md

```diff
@@ -1,42 +1,331 @@
 # User Guide
 
-## Introduction
+# Table of contents
+- [1. Introduction](#1introduction)
+- [2. About this User Guide](#2-about-this-user-guide)
+- [3. Quick Start](#3quick-start)
+- [4. Features](#4features)
+    - [Command Format](#command-format)
+  - [4.1 Help `help`](#41-help-help)
+  - [4.2 Exit the program: `bye`](#42-exit-the-program-bye)
+  - [4.3 HR features `HR`](#43-hr-features-hr)
+    - [Add members: `addMember`](#add-members-addmember)
+    - [Delete members: `delMember`](#delete-members-delmember)
+    - [list members: `listMember`](#list-members-listmember)
+    - [change member information: `changeInfo`](#change-member-information-changeinfo)
+    - [search members: `search`](#search-members-search)
+    - [view contacts of prof/admin: `list prof&admin`](#view-contacts-of-profadmin-list-profadmin)
+    - [view contacts of connections: `list connections`](#view-contacts-of-connections-list-connections)
+  - [4.4 Event features`event`](#44-event-featuresevent)
+    - [Add an event: `addEvent`](#add-an-event-addevent)
+    - [Delete an event: `delEvent`](#delete-an-event-delevent)
+    - [Clear all events: `delEvent all`](#clear-all-events-delevent-all)
+    - [List events: `listEvent`](#list-events-listevent)
+    - [Countdown events `countdown`](#countdown-events-countdown)
+    - [Mark an event as completed `done`](#mark-an-event-as-completed-done)
+    - [Search for an Event `search`](#search-for-an-event-search)
+  - [4.5 Finance features `finance`](#45-finance-features-finance)
+    - [4.5.1 Add finance log entry: `addLog`](#add-finance-log-entry-addlog)
+    - [4.5.2 Delete finance log entry: `delLog`](#delete-finance-log-entry-dellog)
+    - [4.5.3 View financial summary: `summary`](#view-financial-summary-summary)
+- [5. FAQ](#5-faq)
+- [6. Command Summary](#6-command-summary)
+- [7. Glossary](#7-glossary)
 
-{Give a product intro}
+## 1.Introduction
 
-## Quick Start
+Welcome to **CCA Manager!**
+CCA Manager is a revolutionary tool that changes the way you can manage interest groups with unrivaled efficiency and simplicity. Its lightweight Command Line Interface (CLI) allows administrators to breeze through tasks quickly and easily while offering powerful features to advanced users.
 
-{Give steps to get started quickly}
+### 2. About this User Guide 
```

Comment:
I'm not sure if the problem is on my computer, but on GitHub pages, the 'Setting Up' section comprises of only one sentence, 'Refer to the guide here.' with no hyperlink or any other prompts when I mouse over the words.